components-sdk/src/locales/es.json

```
@@ -0,0 +1,63 @@
+{
+  "components.content": "Texto",
+  "components.container": "Contenedor",
+  "components.image": "Imagen",
+  "components.file": "Archivo",
+  "components.separator": "Separador",
+  "components.button": "Botón",
+  "components.button-link": "Botón de enlace",
+  "components.select-menu": "Menú de selección",
+  "components.thumbnail": "Miniatura",
+  "components.text-input": "Entrada de texto",
+  "button.add-component": "Agregar componente",
+  "button.add-inline": "Añadir componente debajo",
+  "button.add-content": "Añadir contenido",
+  "button.add-button": "Añadir botón",
+  "button.new-option": "Nueva opción",
+  "error": "Error:",
+  "actions.add-action": "Añadir acción",
+  "file.upload-file": "Click para subir un archivo",
```

```suggestion
  "file.upload-file": "Subir un archivo",
```

More inline with how it's usually found in the web and the German translation